test/selenium/specs/tc_new.rb

```diff
@@ -10,6 +10,7 @@ class TestNew < Test::Unit::TestCase
   def setup
     page.driver.browser.get(@@privly_test_set[0][:url])
     login(@@privly_test_set[0][:content_server])
+    page.driver.browser.manage.timeouts.implicit_wait = 3
```

Comment:
On my local machine, the tests sometimes fail when the implicit_wait is not mentioned.